Formula/tinyscheme.rb

```diff
@@ -20,35 +22,38 @@ def install
     share.install("init.scm")
     bin.install("scheme")
   end
+
+  test do
+    assert_match "Usage: tinyscheme", shell_output("#{bin}/scheme -?", 1)
```

Comment:
Sorry, I overlooked that information. It's a bit concerning that something as simple would fail on 10.9 and makes me think the resulting binary could be broken, but I'm not sure how feasible it is to debug this failure. (Though my take is that if a perfectly reasonable test fails, then the required action is to fix<sup>#</sup> the root cause revealed by the test instead of adapting the test to the failure.)

<sup>#</sup>) If possible. Or boneyard the formula or do something else, but not weaken the test.